### ISSUE TITLE
Cannot compile component with AOT enabled

### ISSUE THREAD

**Issue by bendem** [albertnadal/ng2-daterange-picker#6]
We've been using your component but when trying to deploy it to prod we hit this error because you're using private fields in your component.

```
ERROR in ng://<project>/node_modules/ng2-daterange-picker/src/ang.daterange.picker.component.html (14,50): Property 'startDateText' is private and only accessible within class 'DaterangePickerComponent'.
ERROR in ng://<project>/node_modules/ng2-daterange-picker/src/ang.daterange.picker.component.html (24,50): Property 'endDateText' is private and only accessible within class 'DaterangePickerComponent'.
ERROR in ng://<project>/node_modules/ng2-daterange-picker/src/ang.daterange.picker.component.html (49,25): Property 'startDate' is private and only accessible within class 'DaterangePickerComponent'.
ERROR in ng://<project>/node_modules/ng2-daterange-picker/src/ang.daterange.picker.component.html (55,25): Property 'endDate' is private and only accessible within class 'DaterangePickerComponent'.
ERROR in ng://<project>/node_modules/ng2-daterange-picker/src/ang.daterange.picker.component.html (14,32): Property 'startDateText' is private and only accessible within class 'DaterangePickerComponent'.
ERROR in ng://<project>/node_modules/ng2-daterange-picker/src/ang.daterange.picker.component.html (24,32): Property 'endDateText' is private and only accessible within class 'DaterangePickerComponent'.
ERROR in ng://<project>/node_modules/ng2-daterange-picker/src/ang.daterange.picker.component.html (24,17): Property 'startDateText' is private and only accessible within class 'DaterangePickerComponent'.
ERROR in ng://<project>/node_modules/ng2-daterange-picker/src/ang.daterange.picker.component.html (24,17): Property 'endDateText' is private and only accessible within class 'DaterangePickerComponent'.
ERROR in ng://<project>/node_modules/ng2-daterange-picker/src/ang.daterange.picker.component.html (35,27): Property 'startDate' is private and only accessible within class 'DaterangePickerComponent'.
ERROR in ng://<project>/node_modules/ng2-daterange-picker/src/ang.daterange.picker.component.html (49,25): Property 'startDate' is private and only accessible within class 'DaterangePickerComponent'.
ERROR in ng://<project>/node_modules/ng2-daterange-picker/src/ang.daterange.picker.component.html (53,25): Property 'endDate' is private and only accessible within class 'DaterangePickerComponent'.
ERROR in ng://<project>/node_modules/ng2-daterange-picker/src/ang.daterange.picker.component.html (53,25): Property 'endDate' is private and only accessible within class 'DaterangePickerComponent'.
ERROR in ng://<project>/node_modules/ng2-daterange-picker/src/ang.daterange.picker.component.html (55,25): Property 'startDate' is private and only accessible within class 'DaterangePickerComponent'.
ERROR in ng://<project>/node_modules/ng2-daterange-picker/src/ang.daterange.picker.component.html (59,25): Property 'endDate' is private and only accessible within class 'DaterangePickerComponent'.
```

The fix is to set `startDate`, `endDate`, `startDateText` and `endDateText` to public.

**Comment by albertnadal** [comment]
Hi bendem, thank you for reporting this issue. I uploaded the fix right now and published the patch version 1.0.6 to the npmjs repository https://www.npmjs.com/package/ng2-daterange-picker Please feel free to report any bug, suggestion or comment. Please help to promote the project by Starring this repository on GitHub :) thank you!

**Comment by bendem** [comment]
Thanks a bunch for the fast reaction time!
